configure.ac

```diff
@@ -1,7 +1,8 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.59)
-AC_INIT([PROJ.4 Projections], 5.0.0, [https://github.com/OSGeo/proj.4/issues], proj)
+AC_INIT([PROJ], [5.0.0],
+        [https://github.com/OSGeo/proj.4/issues], proj)
```

Comment:
As per [this](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Initializing-configure.html) you could also add the project website at the end of the `AC_INIT` call, so it becomes:

```
AC_INIT([PROJ], [5.0.0],
        [https://github.com/OSGeo/proj.4/issues], proj, http://proj4.org)
```